save_discard_button/static/src/scss/indicator_button.scss

```diff
@@ -0,0 +1,64 @@
+.o_form_status_indicator_buttons {
+    margin-left: 10px;
+    .o_form_button_save,
+    .o_form_button_cancel {
+        width: 100%;
+        &.o_form_button_save {
+            background-color: $primary;
+            &:hover {
+                filter: brightness(0.9);
+            }
+        }
+        &.o_form_button_cancel {
+            margin-left: 5px;
+            &:hover {
+                background-color: #f6f7fa;
+                border-color: #e0e2e6;
+            }
+        }
+        .fa-cloud-upload {
+            width: 100%;
+            &:before {
+                content: "Save";
+                font-weight: 500;
+                color: white;
+                display: inline-block;
+                text-align: center;
+                vertical-align: middle;
+                user-select: none;
+                background-color: transparent;
+                border: 1px solid transparent;
+                padding: 0.3rem 2px;
+                font-size: 1.08333333rem;
+                line-height: 1.5;
+                border-radius: 0;
+                transition: none;
+                width: 100%;
+                text-transform: uppercase;
+                font-family: "ubuntu";
+            }
+        }
+        .fa-undo {
+            width: 100%;
+            &:before {
+                content: "Discard";
```

Comment:
Is this translatable?

Reply:
This is not translatable, because in css not allowed to translate.